website/docs/r/pki_secret_backend_acme_eab.html.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: "vault"
+page_title: "Vault: vault_pki_secret_backend_acme_eab resource"
+sidebar_current: "docs-vault-resource-pki-secret-backend-acme-eab"
+description: |-
+  Sets the EST configuration on a PKI Secret Backend for Vault.
+---
+
+# vault\_pki\_secret\_backend\_acme_eab
+
+Allows creating ACME EAB (External Account Binding) tokens and deleting unused ones.
+
+## Example Usage
+
+```hcl
+resource "vault_mount" "test" {
+  path        = "pki"
+  type        = "pki"
+  description = "PKI secret engine mount"
+}
+
+resource "vault_pki_secret_backend_acme_eab" "test" {
+  backend = vault_mount.test.path
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `namespace` - (Optional) The namespace of the target resource.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault/index.html#namespace).
+  *Available only for Vault Enterprise*.
+
+* `backend` - (Required) The path to the PKI secret backend to
+  create the EAB token within, with no leading or trailing `/`s.
+
+* `issuer` - (Optional) Create an EAB token that is specific to an issuer's ACME directory.
+
+* `role` - (Optional) Create an EAB token that is specific to a role's ACME directory.
+
+**NOTE**: EAB tokens are specific to an ACME directory. Within Vault ACME there are different
```

Comment:
I am not very familiar with the ACME terminology, but maybe the sentence can be simplified to 
> Within Vault there are different ACME directories.

And maybe end the sentence with semicolons?

Reply:
Tweaked it a bit.